src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -15,15 +17,31 @@
 
 #include <boost/test/unit_test.hpp>
 
-BOOST_FIXTURE_TEST_SUITE(validation_chainstatemanager_tests, TestingSetup)
+struct ChainTestingSetup : public BasicTestingSetup {
```

Comment:
Any reason to add this here and not to setup_common? Moving it to setup_common would allow removing the confusing `TestingAppInitSequence` declarations in the header. Also, it would allow other tests to use the `ChainTestingSetup`.